openfl/component/director/director.py

```diff
@@ -6,50 +6,282 @@
 import asyncio
 import logging
 import time
-import typing
 from collections import defaultdict
 from pathlib import Path
+from typing import Iterable
+from typing import List
+from typing import Union
 
+from openfl.component import Aggregator
 from openfl.federated import Plan
 from openfl.protocols import director_pb2
+from openfl.transport import AggregatorGRPCServer
 from openfl.utilities.workspace import ExperimentWorkspace
 
 logger = logging.getLogger(__name__)
 
 ENVOY_HEALTH_CHECK_PERIOD = 60  # in seconds
 
 
+class Status:
+    """Experiment's statuses."""
+
+    PENDING = 'PENDING'
+    FINISHED = 'FINISHED'
+    IN_PROGRESS = 'IN_PROGRESS'
+    FAILED = 'FAILED'
+
+
+class Experiment:
```

Comment:
This class should be put in a separate module probably.
My vision is we put it, along with `EnvoyRepresentation` and probably other abstractions under `representations` folder/package. The alternative name may be `common_structures` or something in similar fashion

Reply:
Was moved to `openfl/components/expeiment/experiment.py`